qir/qat/Commandline/ConfigBind.hpp

```diff
@@ -23,8 +25,9 @@ class OpcodeSet;
 template <typename T> class ConfigBind : public IConfigBind
 {
   public:
-    using Type      = typename std::decay<T>::type;
-    using StringSet = std::unordered_set<std::string>;
+    using Type       = typename std::decay<T>::type;
+    using StringSet  = std::unordered_set<std::string>;
+    using StringList = std::vector<std::string>;
 
```

Comment:
See the GitHub Warning for the green line 80 below - typo "configurati**no**".

Reply:
Also search for "serializa**t**ble".

Reply:
Fixed.